finagle-core/src/main/scala/com/twitter/finagle/dispatch/ServerDispatcher.scala

```diff
@@ -1,11 +1,36 @@
 package com.twitter.finagle.dispatch
 
 import com.twitter.finagle.context.Contexts
+import com.twitter.finagle.tracing.{Annotation, Trace}
 import com.twitter.finagle.transport.Transport
 import com.twitter.finagle.{Service, NoStacktrace, CancelledRequestException}
 import com.twitter.util._
 import java.util.concurrent.atomic.AtomicReference
 
+/**
+  * A base Server Dipatcher type that allows us to safely Annotate a
+  * dispatcher via
+  * [[com.twitter.finagle.dispatcher.ServerDispatherAnnotator ServerDispatcherAnnotator]]
+ */
+trait ServerDispatcher[Req, Rep, In, Out] {
```

Comment:
Yeah, it's not. Same as above - I had `yy` that definition from another implementation that was an abstract class that need a Transporter.

Fixed.